src/content/loaders/index.md

```diff
@@ -51,6 +51,8 @@ Loaders are activated by using `loadername!` prefixes in `require()` statements,
 - [`handlebars-loader`](https://github.com/pcardune/handlebars-loader) Compiles Handlebars to HTML
 - [`markup-inline-loader`](https://github.com/asnowwolf/markup-inline-loader) Inline SVG/MathML files to HTML. It’s useful when applying icon font or applying CSS animation to SVG.
 - [`twig-loader`](https://github.com/zimmo-be/twig-loader) Compiles Twig templates and returns a function
+- [`remark-loader`](https://github.com/webpack-contrib/remark-loader) Load markdown through `remark` with built-in image resolution
```

Comment:
Just note - it is official loaders, so we should put them in left column here

Reply:
gotcha will add